src/Aurora.Infrastructure/SearchScraperCollector.cs

```diff
@@ -3,18 +3,20 @@
 using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using Aurora.Infrastructure.Scrapers;
 
 namespace Aurora.Infrastructure
 {
     public class SearchScraperCollector : ISearchScraperCollector
     {
-        private static Dictionary<string, Type> _websiteNameToScraperType = new Dictionary<string, Type>
+        private static readonly Dictionary<string, Type> _websiteNameToScraperType = new()
         {
-            //initialize scraper type linked to proper website
+            { "pornhub", typeof(PornhubScraper) }
```

Comment:
what's better here to use: pornhub name or link to pornhub home page?

Reply:
An enum for supported websites